test/re/frida/MethodTest.java

```diff
@@ -230,6 +264,13 @@ public void tearDown() throws IOException {
     }
 }
 
+class Reflector {
+
+    public static void reflected () {
+        return;
+    }
+}
```

Comment:
Helper classes below should follow the same order as they're used, so given the test is last this one should also be last.

Reply:
ok, I will also remove the space between reflected and ()